packages/jbrowse-cli/README.md

```diff
@@ -120,6 +122,32 @@ EXAMPLES
   $ jbrowse add-assembly GRCh38.config.json
 ```
 
+## `jbrowse create LOCALPATH [VERSION]`
+
+Downloads and installs the latest JBrowse 2 release
+
+```
+USAGE
+  $ jbrowse create LOCALPATH [VERSION]
+
+ARGUMENTS
+  LOCALPATH  Location where JBrowse 2 will be installed
+  VERSION    Version of JBrowse to download, defaults to latest
```

Comment:
I think maybe version should be a flag (although we can't call it `--version`, so it doesn't get confused with reporting the version) instead of an arg.